apis/refs/v1beta1/projectref.go

```diff
@@ -189,6 +189,29 @@ func ResolveProjectID(ctx context.Context, reader client.Reader, obj *unstructur
 	return "", fmt.Errorf("cannot find project id for %v %v/%v", obj.GetKind(), obj.GetNamespace(), obj.GetName())
 }
 
+func ResolveProjectNumber(ctx context.Context, reader client.Reader, key types.NamespacedName) (string, error) {
```

Comment:
The TagBinding uses the project number as its parent and composes the ID from it.